.ci/teamcity/checks/stylelint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "$(dirname "${0}")/../util.sh"
+
+checks-reporter-with-killswitch "Lint: stylelint" \
+  node scripts/stylelint '**/*.s+(a|c)ss'
```

Comment:
Since this is the default argument we shouldn't need to specify it right?

Reply:
Good catch, fixed with 169770f